docs/registry/create/_index.md

```diff
@@ -65,6 +65,10 @@ If you want to write a module to extend support to a new type of component or se
 Most module use cases, however, benefit from extending an existing API, as covered below.
 {{% /alert %}}
 
+{{% alert title="Tip" color="tip" %}}
+If you would like to test your module locally against its intended target platform before uploading it, you can follow the steps for [Iterative module development](/registry/advanced/iterative-development/) to verify that any code changes you have made work as expected on your target platform.
```

Comment:
This does seem a bit uneccessary bc it's shouted out 2 times in appropriate places + I don't think they would know what this means at this point in the process? Like they're not ready to upload the module. Reviewers LMK what you think

Reply:
I don't think it hurts to have it here; but I think better might be a "Test your module" section before the "Deploy your module" section.  But that could be a separate ticket.

Reply:
I'm good with removing with it addressed in ticket as that aligns with matt's advice as well. Thanks andrew!